loom/script/serialize/lsBinWriter.cpp

```diff
@@ -565,7 +565,7 @@ void BinWriter::writeAssembly(const char *sjson, int jsonSize)
     json_error_t jerror;
     json_t       *json = json_loadb(sjson, jsonSize, 0, &jerror);
 
-    lmAssert(json, "Error loading Assembly json: %s\n %s %i\n", jerror.source, jerror.text, jerror.line);
+    lmAssert(json, "Error loading Assembly json: %s\n %s %i\n>>>%s<<<", jerror.source, jerror.text, jerror.line, sjson);
```

Comment:
It shows the JSON it was trying to load (e.g. you can see if it was empty or some random garbage or just invalid json).